OneCommunityGlobal/HighestGoodNetworkApp
# Ramya Lint fix  Add Userprofile - Userprofile subcomponent

![200w](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/89441435/9e6e5487-f69c-4093-a894-301eda0797ff)


# Description

- Lint fix for the AddUserProfile (sub module) of the UserProfile Component

## Main changes explained:

- Commented out print statements ( Will remove it once the PR is ready to merge )
- Enforced a function type for function components
- Fixed variable names to adhere to camel case 
- Removed unused variables and functions 
- Enforced a convention in module import order
- Reordered the function definitions order
- Un-nested ternary expressions
- Require default cases in switch statements

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3.  Verify if the application works as expected with respect to Adding UserProfile 
4. Open the UserProfile  Bar component in your editor and report if you see possible error / warnings .



## Note:
Thoroughly test the Add User new User functionality  and report any error you see so that I can go ahead make similar changes in the  other submodules of UserProfile component
